packages/components/src/Menu/Menu.css

```diff
@@ -80,8 +80,23 @@
   padding: var(--space-small);
 }
 
+:global(.jobber-retheme) .sectionHeader {
+  display: flex;
+  align-items: center;
+  min-height: var(--space-largest);
```

Comment:
To match SG-1 can drop the min height here, and instead use `padding: 12px 8px;`